templates/manifest.json

```diff
@@ -200,6 +200,22 @@
                 "memoryMbytes": 2048,
                 "timeoutSecs": 3600
             }
+        },
+        {
+            "id": "js-cypress",
+            "name": "project_cypress",
+            "label": "Cypress",
+            "category": "javascript",
+            "technologies": [
+                "nodejs"
+            ],
+            "description": "Empty starter actor. A blank slate with minimal starting code for developing actors from scratch using JavaScript.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/feat/add-cypress-actor-template/dist/templates/js-cypress.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 2048,
```

Comment:
Would change this to something higher that works fast enough for the tests to pass.

But probably should be in the free plan limits, so 4GB